notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1158 @@
+{
```

Comment:
<p>A few things here:</p><ul><li>I wonder if you can install openjdk via <code>apt-get</code>, i.e <code>sudo apt-get install openjdk-8-jdk</code> ?</li><li>If the above doesn't work, can you please convert the Java paths into a couple variables and fix throughout?</li><li class="ql-indent-1"><code>JAVA_RELEASE = "8u332-b09"</code></li><li class="ql-indent-1"><code>JAVA_INSTALLATION = f"openlogic-openjdk-{JAVA_RELEASE}-linux-x64</code></li><li>Can you please pin the <code>pyspark</code> installation to match the Spark version <a href="https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-release-2.0" target="_blank">latest Dataproc release</a>? I believe this is 3.1.3. Please add a comment mentioning that the version is pinned for this reason.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I'm not sure about the testing environment, but in my local, <code>sudo apt-get</code> doesn't work inside of the notebook due to the root password.</p>